.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+| Q | A
+| --- | ---
+| Bug? | no
+| Version Used | Specific tag or commit sha
+| New Feature? | no
+
+#### Expected Behavior
+
+Briefly describe the behavior you expect (bug) or you want (feature request).
+
+#### Actual Behavior
+
+Describe the actual behavior, which error occurs or how the system currently behaves. 
+
+#### Steps to Reproduce
+
+1. Please collect the exact steps to reproduce the bug, don't hesitate to add code examples or screenshots.
```

Comment:
Let's simplify this instruction:

"What are the steps to reproduce this bug? Please add code examples, screenshots or links to GitHub repositories that reproduce the problem."